addons/website_event_meet/views/event_meet_templates_list.xml

```diff
@@ -117,24 +117,32 @@
         t-attf-class="card o_wevent_meeting_room_card w-100 my-2 bg-light d-block text-decoration-none rounded-0"
         t-att-href="meeting_room_href"
         t-att-data-toggle="meeting_room_data_toggle"
-        t-att-data-target="meeting_room_data_target">
+        t-att-data-target="meeting_room_data_target"
+        t-att-data-publish="meeting_room.website_published and 'on' or 'off'">
```

Comment:
What is the purpose of this attribute ? Does not seem used ...

Reply:
This attribute is responsible for reducing opacity of the unpublished objects (css is defined in website for generic use, and it's already used in modules like `website_sale`, `website_hr_recruitment` etc) so we utilized the same instead of introducing special class for opacity.

Reply:
Ok :)